src/data-table/data-table.styles.tsx

```diff
@@ -0,0 +1,177 @@
+                td:first-child {
+                    padding-left: 1.5rem;
+                }
+            `;
+        }
+    }}
+    ${(props) => {
+        if (props.$addMarginToLastColumn) {
+            return css`
+                th:last-child,
+                td:last-child {
+                    padding-right: 1.5rem;
+                }
+            `;
+        }
+    }}
+`;
+
+export const HeaderRow = styled.tr`
+    background-color: #f5f5f5;
+    height: 5rem;
+    border-bottom: 0.125rem solid ${Color.Neutral[6]};
+`;
+
+export const HeaderCell = styled.th<HeaderCellProps>`
+    padding: 1rem 0;
+    text-align: left;
+    cursor: ${(props) => (props.$clickable ? "pointer" : "default")};
+    max-width: ${(props) => (props.$maxWidth ? props.$maxWidth : "auto")};
+    vertical-align: middle;
+    ${TextStyleHelper.getFontFamily("H5", "bold")}
+    color: ${Color.Neutral[1]}
+`;
+
+export const HeaderCellWrapper = styled.div`
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+
+    svg {
+        color: ${Color.Neutral[1]};
+        margin-left: 0.5rem;
+    }
+`;
+
+export const BodyRow = styled.tr<BodyRowProps>`
+    background-color: ${(props) => {
+        if (props.$isSelected) {
+            return css`
+                ${Color.Accent.Light[5]};
+            `;
+        } else if (props.$alternating) {
+            return css`
+                ${Color.Neutral[7]};
+            `;
+        } else {
+            return css`
+                ${Color.Neutral[8]};
+            `;
+        }
+    }};
+    border-top: 0.125rem solid ${Color.Neutral[6]};
+    &:hover {
+        background-color: ${(props) => {
+            if (!props.$isSelected && props.$isSelectable) {
+                return css`
+                    ${Color.Accent.Light[4]};
+                `;
+            }
+        }};
+    }
+    &:first-child {
+        border-top: none;
+    }
+`;
+
+export const BodyCell = styled.td<BodyCellProps>`
+    padding: 1.25rem 1rem;
+    width: ${(props) => (props.$width ? props.$width : "auto")};
+    vertical-align: middle;
+`;
+
+export const CheckBoxWrapper = styled.div`
+    display: flex;
+    align-items: center;
+    justify-content: center;
+`;
+
+export const CheckBox = styled(Checkbox)``;
+
+export const LoaderWrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+`;
+
+export const EmptyDataElement = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 2rem;
+    opacity: 0;
+    img {
+        width: 25rem;
+        height: auto;
+        flex: none;
+        order: 0;
+        flex-grow: 0;
+    }
+    animation: fade-in 1s ease-in forwards;
+    @keyframes fade-in {
+        100% {
+            opacity: 1;
+        }
+    }
+`;
+
+export const CustomErrorDisplay = styled(ErrorDisplay)`
+    div {
+        h1 {
+            ${TextStyleHelper.getFontFamily("H3", "bold")}
+        }
+    }
```

Comment:
not ideal way to style the text elements. let's clarify with UX if h3 is the standard size for an inline error display. we can introduce a prop like `displayType: "default" | "small"` as part of the follow up PR.

Reply:
I'll remove the font size and weight change. Follow up URL, I'll get update from UX and do the changes.